packages/react-native-platform-override/src/PathUtils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as path from 'path';
+
+/**
+ * Convert a path to use Unix-style slashes
+ */
+export function unixPath(filepath: string): string {
+  return filepath.replace(/\\/g, '/');
```

Comment:
>\\ [](start = 27, length = 2)

In Unix backslash is used for whitespaces. Could we get into a situation that this code corrupts valid Unix paths?

Reply:
I'll see if I can add some better logic here